invenio_sipstore/models.py

```diff
@@ -155,6 +175,31 @@ def validate_key(self, filepath, filepath_):
     """Relation to the SIP along which given file was submitted."""
 
 
+class SIPMetadata(db.Model, Timestamp):
+    """Extra SIP info regarding metadata."""
+
+    __tablename__ = 'sipstore_sipmetadata'
+
+    id = db.Column(UUIDType, primary_key=True, default=uuid.uuid4)
+    """Id of metadata."""
+
+    sip_id = db.Column(UUIDType,
+                       db.ForeignKey(SIP.id, name='fk_sipmetadata_sip_id'))
+    """Id of SIP."""
+
+    format = db.Column(db.String(7), nullable=False)
```

Comment:
should this be longer string, given that 7 char string would save us anything w.r.t. to implementation in the DB?